.env.example

```
@@ -1,106 +1,254 @@
+
+# X.AI (Elon Musk's company)
+# Get your API key from: https://console.x.ai/
+XAI_API_KEY=your_xai_api_key_here
+
+# Moonshot AI (Kimi models)
+# Get your API key from: https://platform.moonshot.ai/console/api-keys
+MOONSHOT_API_KEY=your_moonshot_api_key_here
+
+# Hugging Face
+# Get your API key from: https://huggingface.co/settings/tokens
+HuggingFace_API_KEY=your_huggingface_api_key_here
```

dotenv-linter (3.3.0)

[warning] [LowercaseKey] The HuggingFace_API_KEY key should be in uppercase

(LowercaseKey)